src/plasoscaffolder/frontend/controller/sqlite_controller.py

```diff
@@ -1,34 +1,35 @@
 # -*- coding: utf-8 -*-
-"""File representing the controller for SQLite plugin."""
+"""File representing the Controller for SQLite plugin."""
+import os
+
 import click
-from plasoscaffolder.bll.mappings.formatter_mapping import FormatterMapper
-from plasoscaffolder.bll.mappings.init_mapping import InitMapper
-from plasoscaffolder.bll.mappings.mapping_helper import MappingHelper
-from plasoscaffolder.bll.mappings.parser_mapping import ParserMapper
-from plasoscaffolder.bll.services.sqlite_generator import SQLiteGenerator
-from plasoscaffolder.bll.services.sqlite_plugin_helper import SQLitePluginHelper
-from plasoscaffolder.bll.services.sqlite_plugin_path_helper import \
-  SQLitePluginPathHelper
-from plasoscaffolder.common.base_output_handler import BaseOutputHandler
-from plasoscaffolder.common.file_handler import FileHandler
-from plasoscaffolder.common.output_handler_click import OutputHandlerClick
+from plasoscaffolder.bll.mappings import formatter_mapping
+from plasoscaffolder.bll.mappings import init_mapping
+from plasoscaffolder.bll.mappings import mapping_helper
+from plasoscaffolder.bll.mappings import parser_mapping
+from plasoscaffolder.bll.services import sqlite_generator
+from plasoscaffolder.bll.services import sqlite_plugin_helper
+from plasoscaffolder.bll.services import sqlite_plugin_path_helper
+from plasoscaffolder.common import base_output_handler
+from plasoscaffolder.common import file_handler
+from plasoscaffolder.common import output_handler_click
 
 
 class SQLiteController(object):
-  """Class representing the controller for the SQLite controller."""
+  """Class representing the Controller for the SQLite Controller."""
 
-  def __init__(self, output_handler: BaseOutputHandler):
+  def __init__(self, output_handler: base_output_handler.BaseOutputHandler()):
     super(SQLiteController, self).__init__()
-    self.path = None
-    self.name = None
-    self.testfile = None
-    self.events = None
-    self.plugin_helper = SQLitePluginHelper()
-    self.output_handler = output_handler
+    self.__path = None
+    self.__name = None
+    self.__testfile = None
+    self.__events = None
+    self.__plugin_helper = sqlite_plugin_helper.SQLitePluginHelper()
+    self.__output_handler = output_handler
 
-  def source_path(self, ctx: click.core.Context, param: click.core.Option,
-      value: str) -> str:
-    """Saving the source path.
+  def SourcePath(self, _ctx: click.core.Context, _param: click.core.Option,
```

Comment:
i used the _ because pylint now doesn't give me an error. that i didn't use ctx and param. But i do need them because the click callback requires them. 
Should i remove them and live with the pylint error message?